tracker/static/style.css

```diff
@@ -91,6 +91,7 @@ select {
 	list-style: outside none none;
 	margin: -0.5em 0;
 	padding: 0;
+	overflow: hidden;
```

Comment:
why do we want to hide overflows here and where do we expect an element to overflow?

Reply:
I wanted to re-use the pagination macros that already existed. Those however is contained in a div which moves it to the next line. This is no problem on the Log page where it comes from, but in the issues there is "vunlerable | all" on the left-hand side. I wanted to keep the pagination on the same line which is why i floated it to right end hence needed an overflow: hidden in the parent container.